packages/keyring-controller/src/KeyringController.ts

```diff
@@ -251,11 +232,9 @@ export type KeyringControllerOptions = {
 );
 
 /**
- * @type KeyringObject
+ * KeyringObject
  *
  * Keyring object to return in fullUpdate
- * @property type - Keyring type
```

Comment:
Nit: While `@property` was certainly not the proper way to document type properties (it's not supported by TSdoc, TypeDoc, or VSCode), we can document them like this:

```typescript
export type KeyringObject = {
  /**
   * Associated accounts
   */
  accounts: string[];
  /**
   * Keyring type
   */
  type: string;
};
```

It is nice to have docs for all properties, certainly if they're already documented. I see a few others were deleted in this PR as well

Reply:
Good idea, I added some descriptions on `KeyringObject` and `KeyringControllerState`: [1334e78](https://github.com/MetaMask/core/pull/5170/commits/1334e78782775c9d0261a429b90141f72672af93)